src/spec/src/browser.ts

```diff
@@ -408,6 +400,8 @@ export class Spec implements SpecLike<Spec> {
       }
     }
 
+    const regs = Object.entries(this.options.registries)
+    regs.push(['npm', this.options.registry])
```

Comment:
what if `this.options.registries.npm` is already defined? should there be a condition here for that?

Reply:
No. If it's already defined, it'll come before this one, and this will be a no-op. It's only relevant if no other setting has been defined.

Reply:
But may as well have a guard against the unnecessary push, I guess. Updated.